languages/python/concepts/string-formatting/about.md

```diff
@@ -0,0 +1,141 @@
+## String Formatting in Python
+
```

Comment:
```suggestion
The [Zen of Python][zen-of-python] asserts there should be “one _obvious_ way to do something in Python".  But when it comes to string formatting, things are a little .... _less zen_.   It can be surprising to find out that there are **four** major ways to perform string formatting in Python - each for a different scenario.
```